components/jms-mq-support/org.wso2.carbon.cache.sync.jms.manager/IBMMQ_README.md

```diff
@@ -0,0 +1,54 @@
+#### Configuring IBM MQ
+
+* Setup IBM MQ using their official documentation. 
+  * You may follow https://developer.ibm.com/learningpaths/ibm-mq-badge/create-configure-queue-manager
+  * **Troubleshooting**: 
+    * If you are using MAC with Apple chip, you may need to create the docker image yourself. Refer this link for 
+      more information. 
+      https://community.ibm.com/community/user/integration/blogs/richard-coppen/2023/06/30/ibm-mq-9330-container-image-now-available-for-appl 
+    * When building the docker, the IBM MQ version is defined by [this config](https://github.com/ibm-messaging/mq-container/blob/master/config.env#L4). The given version should be available in [this 
+      directory](https://public.dhe.ibm.com/ibmdl/export/pub/software/websphere/messaging/mqadv/) for download. 
+      Otherwise, the image creation will fail.
+    * If you are using Rancher, the IBM MQ console might not be accessible over 9443 port mapping.
+      Hence, use 9243:9443 port mapping when creating the docker container. 
+  * Make sure to configure the channel authentication. For the ease of use, run the following two commands to disable 
+    channel authentication.
+    
+    `runmqsc QM1`
+
+     `ALTER QMGR CHLAUTH(DISABLED)`
+
+     `REFRESH SECURITY TYPE(CONNAUTH)`
+  * Create a topic for the cache invalidation syncing and make sure the user who’s accessing the JMS application in 
+    the next steps will have permission to publish and subscribe to the topic.
+  * Create a `.bindings` file for the created topic and save it in a desired location in your computer. You need this 
+    when configuring the Identity Server (You can refer [this document](https://ei.docs.wso2.com/en/latest/micro-integrator/setup/brokers/configure-with-IBM-websphereMQ/#generating-the-bindings-file) on creating the .binding file)
+
+#### Configure the WSO2 IS
+
+1. Follow the instructions in the [IS_README.md](IS_README.md)
+2. Then add the following config(update it with your values) in the deployment.toml file
+  ```
+  [cache_invalidator.mb]
+  enabled="true"
+  broker_type="jms"
+  initial_naming_factory="com.sun.jndi.fscontext.RefFSContextFactory"
+  provider_url="file:/Documents/conf/jndi"
+  topic_name="this_not_considered_for_ibm"
```

Comment:
what would happen if we didn't specify this config then? if this is not handled properly shall we improve the logic

Reply:
This value is not considered at all when creating the connection. Only the value given via the config file is considered. So there's no errors coming from the connector side.